src/components/PrintHandler/PrintHandler.js

```diff
@@ -14,8 +15,13 @@ const PrintHandler = () => {
     shallowEqual,
   );
 
+  let containerClass = 'PrintHandler';
+  if (isIOS) {
+    containerClass += ' ios-print';
+  }
+
   return isDisabled ? null : (
-    <div className="PrintHandler">
+    <div className={containerClass}>
```

Comment:
Can we use the classnames package please ;). So it will be like
```javascript
import classNames from 'classnames';

<div className={classNames({
  PrintHandler: true,
  'ios-print': isIOS,
})}>
...
</div>
```